packages/edit-widgets/src/blocks/widget-area/block.json

```diff
@@ -15,7 +15,8 @@
 		"customClassName": false,
 		"reusable": false,
 		"__experimentalToolbar": false,
-		"__experimentalParentSelector": false
+		"__experimentalParentSelector": false,
+		"__experimentalDisableBlockOverlay": true
```

Comment:
What I'm not 100% sure is this change and the reason it was needed. Riad made a change to when to apply the block overlay in a more generic way([see here](https://github.com/WordPress/gutenberg/pull/41156/files#diff-aa92c48095c35f17feb834dd919baf52274a73c7b1180ff5acce7b6087b126cfR2724)) and this made the special `widget-area` blocks to have an overlay, but they actually contain `panel components` and don't need them.

I thought about reverting those changes back to explicitly checking in individual blocks, but it mostly makes sense to have the overlay for blocks with inner blocks controlled and I considered the `widget area` as a special case. This might cause the same issue in third party blocks that might have implemented something like that..

--cc @mcsf @jorgefilipecosta

Reply:
I'm not super sure, but I guess we can keep this. Have you developed a preference since last week?

Reply:
I'm not a fan for this flag, but it seems like an edge case usage of blocks. It feels better to have a consistent way to handle the overlay for the blocks with controlled inner blocks.

Reply:
I'm thinking of inverting the flag and making it opt-in instead of opt-out because it's better for performance (just a bit)

Reply:
(I'm still hesitant though, we'll see :P)